test/EFCore.PG.FunctionalTests/Query/NorthwindWhereQueryNpgsqlTest.cs

```diff
@@ -170,26 +173,43 @@ public override async Task Where_compare_constructed_multi_value_not_equal(bool
 
     public override async Task Where_compare_tuple_constructed_equal(bool async)
     {
-        //  Anonymous type to constant comparison. Issue #14672.
-        await AssertTranslationFailed(() => base.Where_compare_tuple_constructed_equal(async));
+        await AssertQuery(
+            async,
+            ss => ss.Set<Customer>().Where(c => new Tuple<string>(c.City).Equals(new Tuple<string>("London"))),
+            entryCount: 6);
 
-        AssertSql();
+        AssertSql(
+            @"SELECT c.""CustomerID"", c.""Address"", c.""City"", c.""CompanyName"", c.""ContactName"", c.""ContactTitle"", c.""Country"", c.""Fax"", c.""Phone"", c.""PostalCode"", c.""Region""
+FROM ""Customers"" AS c
+WHERE (c.""City"") = ('London')");
     }
 
     public override async Task Where_compare_tuple_constructed_multi_value_equal(bool async)
     {
-        //  Anonymous type to constant comparison. Issue #14672.
-        await AssertTranslationFailed(() => base.Where_compare_tuple_constructed_multi_value_equal(async));
+        await AssertQuery(
```

Comment:
/cc @smitpatel note these translation failure tests which now pass via row values (resulting SQL is even a bit tighter than the expanded `WHERE c.""City"" = 'Sao Paulo' AND c.""Country"" = 'Brazil'`).

If we decide to bring this to relational, then specifically for SQL Server which doesn't support this, we could have a post-processing step which expands RowValueExpression to the equivalent verbose form (as above).